### PR TITLE
Add basic styling for the landing page

### PR DESCRIPTION
**Pull request by lukaszklis**

### CODE REVIEW

File: templates/pages/index.html.njk [hunk 1]
```diff
@@ -4,22 +4,71 @@
...
+            Our call for speakers is open until December 23, 2018. lorem ipsum dolor sit lalala orem ipsum dolor sit
+            lalala orem ipsum dolor sit lalala orem ipsum dolor sit lalala orem ipsum dolor sit lalalaorem ipsum dolor
+            sit lalala.
+            <a href="{{ contents['call-for-speakers.json'].url }}" class="link--slateblack">Read more</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <div class="section-news">
```
Comment (lukaszklis) [inline templates/pages/index.html.njk]: We should make this dynamic, added it as static text just for the preview.

File: templates/pages/index.html.njk [hunk 2]
```diff
@@ -4,22 +4,71 @@
...
+    </div>
+    <div class="section-video-text ttu mb4 tc">Impressions of JSConf EU 2018</div>
+  </a>
+  <section class="section-cfs">
+    <div class="grid">
+      <div class="grid-item-full">
+        <div class="section-ornament section-ornament--right">
+          <h2 class="heading--massive heading--section">Call() for Speakers</h2>
+          <p class="mr6-l">
+            Our call for speakers is open until December 23, 2018. lorem ipsum dolor sit lalala orem ipsum dolor sit
```
Comment (lukaszklis) [inline templates/pages/index.html.njk]: We need proper copy in here.

File: templates/pages/index.html.njk (hunk 1 above)
Comment (cramforce) [reply templates/pages/index.html.njk]: Please wrap in comments for now. I'll add logic at a later time.

File: templates/pages/index.html.njk (hunk 2 above)
Comment (cramforce) [reply templates/pages/index.html.njk]: Paraphrased from linked page:

We are inviting the JavaScript community to submit talks for the upcoming JSConf EU (June 1 – 2 2019, Berlin). Submit until December 23rd!

**Comment by cramforce** [comment]
I added commits for the video and X in #9. The X still needs positioning within the layout but otherwise everything should be in place.

**Pull request merged**